examples/stargate-jwt-auth/start_jwt.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+export CASSTAG=3.11.8
+export SGTAG=v1.0.7
+export KCTAG=11.0.2
+
+# Make sure cass-1, the seed node, is up before bringing up other nodes and stargate
+docker-compose up -d cass-1
+# Wait until the seed node is up before bringing up more nodes
+(docker-compose logs -f cass-1 &) | grep -q "Created default superuser role"
+
+# Bring up the 2nd C* node
+docker-compose up -d cass-2
+(docker-compose logs -f cass-1 &) | grep -q "is now part of the cluster"
+# Bring up the 3rd C* node
+docker-compose up -d cass-3
+(docker-compose logs -f cass-1 &) | grep -q "is now part of the cluster"
+
+# this is where a cql script must be run to insert data and grant permissions
+docker run --rm -it -v /Users/lorina.poland/CLONES/stargate/docker-images/examples/stargate-jwt-auth/scripts:/scripts  \
+-v /Users/lorina.poland/CLONES/stargate/docker-images/examples/stargate-jwt-auth/cqlshrc:/.cassandra/cqlshrc  \
+--env CQLSH_HOST=host.docker.internal --env CQLSH_PORT=9045  nuvo/docker-cqlsh
+
+# Bring up keycloak for handling JWTs
+docker-compose up -d keycloak
+
```

Comment:
Might be good to setup the user here

```shell
echo "* Request for authorization"
TOKEN=$(curl -s --data "username=admin&password=admin&grant_type=password&client_id=admin-cli" http://localhost:4444/auth/realms/master/protocol/openid-connect/token | jq -r '.access_token')

echo "\n"
echo " * user creation\n"
curl -L -X POST 'http://localhost:4444/auth/admin/realms/stargate/users' \
-H 'Content-Type: application/json' \
-H "Authorization: bearer $TOKEN" \
--data-raw '{
    "username": "testuser1",
    "email": "foo@bar.com",
    "enabled": true,
    "emailVerified": true,
    "attributes": {
        "userid": [
            "9876"
        ],
        "role": [
            "web_user"
        ]
    },
    "credentials": [
        {
            "type": "password",
            "value": "testuser1",
            "temporary": "false"
        }
    ]
}'
```

Reply:
I added this code to the startup script and made a note on the docs page that this step is completed in the startup script. I still show the step in the docs for the explanation.